File: app/src/main/java/dev/ragnarok/fenrir/AccountType.java
package dev.ragnarok.fenrir;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@IntDef({AccountType.BY_TYPE,
        AccountType.VK_ANDROID,
        AccountType.VK_ANDROID_HIDDEN,
        AccountType.KATE,
        AccountType.KATE_HIDDEN})
@Retention(RetentionPolicy.SOURCE)
public @interface AccountType {
    int BY_TYPE = 0;
    int VK_ANDROID = 1;
    int VK_ANDROID_HIDDEN = 2;
    int KATE = 3;
    int KATE_HIDDEN = 4;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/AccountType.kt
package dev.ragnarok.fenrir

import androidx.annotation.IntDef

@IntDef(
    AccountType.BY_TYPE,
    AccountType.VK_ANDROID,
    AccountType.VK_ANDROID_HIDDEN,
    AccountType.KATE,
    AccountType.KATE_HIDDEN
)
@Retention(
    AnnotationRetention.SOURCE
)
annotation class AccountType {
    companion object {
        const val BY_TYPE = 0
        const val VK_ANDROID = 1
        const val VK_ANDROID_HIDDEN = 2
        const val KATE = 3
        const val KATE_HIDDEN = 4
    }
}